storage/scylladb_client.c

```diff
@@ -12,7 +12,7 @@
 static struct db_keyspace_names_s {
   db_client_usage_t usage;
   const char* name;
-} db_keyspace_names[] = {{DB_USAGE_REATTACH, "reattachment"}};
+} db_keyspace_names[] = {{DB_USAGE_REATTACH, "reattachment"}, {DB_USAGE_PERMANODE, "db_chronicle"}};
```

Comment:
I am confused with `DB_USAGE_PERMANODE`. Why did you name it?

Reply:
This enumeration is for `db_client_service` to indicate the usage.
I rename every `chronicle` to `permanode` expect the keyspace name `db_chronicle` since I don't alternate the keyspace name in our running ScyllaDB cluster.